test/journeys/specs/oneOnOne/global/basic.js

```diff
@@ -9,187 +9,176 @@ import {runAxe} from '../../../lib/axe';
 import {elements, openMenuAndClickButton} from '../../../lib/test-helpers/space-widget/main';
 
 describe('Widget Space: One on One', () => {
```

Comment:
Should we add the "Global" text to this main describe block since we removed the child describe?

Reply:
Ah, gotcha